src/components/timeline/Dash.tsx

```diff
@@ -0,0 +1,88 @@
+/*
+ * Draws fully customizable dashed lines vertically or horizontally
+ * source:react-native-dash
+ * copied as a result of 'SyntaxError: Cannot use import statement outside a module' pointing to util.js file
+ */
+
+import React from 'react';
```

Comment:
Yes, we do and it works fine in our public repo. Here I keep getting `error 'SyntaxError: Cannot use import statement outside a module'` and that was the only solution that worked for me.

Reply:
Not sure I understand what you wrote (: 
Did you try using `react-native-dash` in public? 
What threw `error 'SyntaxError: Cannot use import statement outside a module'` exactly?

Reply:
Yes, I tried to use `react-native-dash` but got the above error. Since it's not our library I couldn't find a way to resolve it so I added a Dash component to our lib. I'm thinking to edit it to try and solve the height issue I have with the dashed line.

Reply:
What height issue you have with the dashed line? 
The new Dash component has a dependency on `react-native-measureme` that doesn't seem maintained, have a valid license or do anything we can't do ourselves. Why do you need this package for?

Reply:
Regarding the height, I fixed it on the Timeline side - the Dash component needs height (instead of flex) in this case as we calculate the lines height based on the container layout.
Regarding the `react-native-dash', I just copied the file and installed its dependencies, I didn't look much into it. I just saw it calculating the height and width and returning a "layout aware" view. Do you suggest I'll implement it as well? It's doable I think

Reply:
I just didn't want to have implement a Dash component in public and use react-native-dash in private. 

As for the height, I think there's a small bug related. 
In the example screen you added an expand feature. After expanding the item the line extends, but when collapsing the item, the line stay the same instead of shrinking back.

Reply:
I know, I'm working on that just now (the example is for that). Seems like container's `onLayout` is not called on the previous/known layout (if you collapse it again it won't be called either). Looks like a memo of some sort. Still no clue...

Reply:
Done